### PR TITLE
Helps 1619 merge test hotfix to main

### PR DESCRIPTION
**Pull request by RusGraf**

### CODE REVIEW

File: ldk/javascript/examples/self-test-loop/src/tests/whisper/react-whisper-utils.tsx [hunk 1]
```diff
@@ -74,10 +80,15 @@ function withWrapper(WrappedComponent: React.JSXElementConstructor<TestComponent
 
 export class WhisperTestWrapper {
   onResolve: (value: boolean | PromiseLike<boolean>) => void;
+
```
Comment (BrettMugglinOlive) [inline ldk/javascript/examples/self-test-loop/src/tests/whisper/react-whisper-utils.tsx:83]: Why add the spaces here?

Comment (richseviora) [reply ldk/javascript/examples/self-test-loop/src/tests/whisper/react-whisper-utils.tsx:83]: I think this is a default behaviour in (I want to say) eslint or something

**Review by richseviora — APPROVED** [review]

**Review by BrettMugglinOlive — APPROVED** [review]

File: ldk/javascript/examples/self-test-loop/src/tests/whisper/react-whisper-utils.tsx (hunk 1 above)
Comment (BrettMugglinOlive) [reply ldk/javascript/examples/self-test-loop/src/tests/whisper/react-whisper-utils.tsx:83]: Huh...weird. I wonder why that's the default?

Comment (RusGraf) [reply ldk/javascript/examples/self-test-loop/src/tests/whisper/react-whisper-utils.tsx:83]: Yes, if you open SelfTestLoop project directly in the vscode, it shows much more eslint issues. https://eslint.org/docs/rules/lines-between-class-members

**Pull request merged**